### PR TITLE
update basic tutorials for foxy

### PR DESCRIPTION
**Pull request by mikaelarguedas**
Signed-off-by: Mikael Arguedas <mikael.arguedas@gmail.com>

### CODE REVIEW

File: SROS2_MacOS.md [hunk 1]
```diff
@@ -78,8 +78,9 @@ ros2 security create_keystore demo_keys
 #### Generate keys and certificates for the talker and listener nodes
 
 ```bash
-ros2 security create_key demo_keys /talker
-ros2 security create_key demo_keys /listener
+
```
Comment (ruffsl) [inline SROS2_MacOS.md:81]: nit: no white space needed.

Comment (mikaelarguedas) [reply SROS2_MacOS.md]: :+1: see https://github.com/ros2/sros2/pull/201/commits/2d2ab6deec51dcb6a55d1ce6d9507879aea4b4ce

**Comment by mikaelarguedas** [comment]
> There are some parts of the tutorial that still sound like each Node can have independent security settings, e.g.:

@ivanpauno I'm not sure what the best wording for this would be, we could replace "node" with "process" but apart from that I'm not sure how to add correctness without adding complexity.
Did you have a wording suggestion in mind ?

**Comment by ivanpauno** [comment]
> @ivanpauno I'm not sure what the best wording for this would be, we could replace "node" with "process" but apart from that I'm not sure how to add correctness without adding complexity.
Did you have a wording suggestion in mind ?

Not much more than that, my idea was to replace `node` with `process` everywhere (or almost everywhere).

I think it can be done in a follow-up if you want, as this makes the tutorial work again.

**Comment by mikaelarguedas** [comment]
I gave that an unsatisfactory try so I'll just merge this as is as suggested to make fix the tutorial and revisit the wording at a later date. thanks @ivanpauno for the feedback

**Pull request merged**